src/main/java/com/deque/axe/android/AxeMetaData.java

```diff
@@ -0,0 +1,18 @@
+package com.deque.axe.android;
+
+import com.deque.axe.android.utils.JsonSerializable;
+
+@SuppressWarnings({"FieldCanBeLocal", "unused"})
+class AxeMetaData implements JsonSerializable {
+
+  private final String libVersion = getClass().getPackage().getSpecificationVersion();
+
+  private final String subject;
```

Comment:
`subject` as a field is generic enough to be confusing. Would `applicationIdentifier` work?

Reply:
I took this field name for AXRL. Though, it looks like they've changed it to testSubject. Fuck AXRL. I'm gonna go with "appIdentifier".